interfaces/automated-testing/scd/scd.yaml

```diff
@@ -35,6 +35,402 @@ components:
+        center:
+          $ref: "#/components/schemas/LatLngPoint"
+        radius:
+          $ref: "#/components/schemas/Radius"
+    Volume3D:
+      description: >-
+        A three-dimensional geographic volume consisting of a
+        vertically-extruded shape. Exactly one outline must be specified.
+      type: object
+      properties:
+        outline_circle:
+          anyOf:
+            - $ref: "#/components/schemas/Circle"
+          description: A circular geographic shape on the surface of the earth.
+        outline_polygon:
+          anyOf:
+            - $ref: "#/components/schemas/Polygon"
+          description: >-
+            A polygonal geographic shape on the surface of the earth.
+        altitude_lower:
+          description: >-
+            Minimum bounding altitude of this volume. Must be less than
+            altitude_upper, if specified.
+          anyOf:
+            - $ref: "#/components/schemas/Altitude"
+        altitude_upper:
+          description: >-
+            Maximum bounding altitude of this volume. Must be greater than
+            altitude_lower, if specified.
+          anyOf:
+            - $ref: "#/components/schemas/Altitude"
+    Volume4D:
+      description: Contiguous block of geographic spacetime.
+      required:
+        - volume
+      type: object
+      properties:
+        volume:
+          $ref: "#/components/schemas/Volume3D"
+        time_start:
+          description: Beginning time of this volume. Must be before time_end.
+          anyOf:
+            - $ref: "#/components/schemas/Time"
+        time_end:
+          description: End time of this volume. Must be after time_start.
+          anyOf:
+            - $ref: "#/components/schemas/Time"
+
+    UssBaseURL:
+      description: >-
+        The base URL of a USS implementation of part or all of the USS-USS API.
+        Per the USS-USS API, the full URL of a particular resource can be
+        constructed by appending, e.g., `/uss/v1/{resource}/{id}` to this base
+        URL. Accordingly, this URL may not have a trailing '/' character.
+      type: string
+      example: https://uss.example.com/utm
+
+    UssAvailabilityState:
+      type: string
+      enum:
+        - Unknown
+        - Normal
+        - Down
+      description: >-
+        A USS is presumed to be in the Unknown state absent indication otherwise
+        by a USS with availability arbitration scope.  Upon determination via
+        availability arbitration, a USS is Down when it does not respond
+        appropriately, and a Down USS may not perform the following operations
+        in the DSS:
+        * Create an operational intent in the Accepted or Activated states
+        * Modify an operational intent whose new or unchanged state is Accepted
+        or Activated
+        * Delete an operational intent
+        A USS in the Unknown state possesses all the capabilities, within the
+        DSS, of a USS in the Normal state.
+    EntityOVN:
```

Comment:
This schema is only used by the Key schema which is not needed in the interface. It can be removed.

Reply:
EntityOVN has been removed now, thanks for this.